src/routes/+layout.svelte

```diff
@@ -4,23 +4,27 @@
     import Footer from "$lib/components/Footer.svelte";
     import { browser } from "$app/environment";
 
-    // Google Analytics Script
-    function gtag() {
-        window.dataLayer.push(arguments);
-    }
+    // Google Analytics
     if (browser) {
+        // @ts-ignore
+        window.gtag = function () {
+            window.dataLayer.push(arguments);
+        };
         window.dataLayer = window.dataLayer || [];
-        gtag('js', new Date());
-        gtag('config', 'G-LLGRDWVVEV');
+        gtag("js", new Date());
+        gtag("config", "G-LLGRDWVVEV");
```

Comment:
We need linting rules in this repo pretty bad. I usually like the abnb standard of single quotes for strings. Doesn't matter one way or the other though, just need a consistent and automatically applied style.

Reply:
oh, shit that's my bad i will put them in single quotes just didn't even think about it